texar/data/data/multi_aligned_data.py

```diff
@@ -687,8 +573,7 @@ def text_id_name(self, name_or_id):
         i = self._maybe_name_to_id(name_or_id)
```

Comment:
Implementation for all these `*_name` functions should change, since we don't store `_name_prefix` anymore. `_names` should be used directly.

Reply:
Resolved this in 462312cab452a7f1c3eacce634d65f1a50c2fbbc. `name_prefix` is no more an instance variable of the class but a temporary variable in the constructor needed to construct `self._name_to_id` dict. We now use `self._names` in all the getters.